client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.js

```diff
@@ -142,7 +142,16 @@ function CheckoutSummaryFeaturesList() {
 
 function SupportText( { hasPlanInCart, isJetpackNotAtomic } ) {
 	const translate = useTranslate();
+	const plan = usePlanInCart();
+
 	if ( hasPlanInCart && ! isJetpackNotAtomic ) {
+		if (
+			'personal-bundle' === plan.wpcom_meta?.product_slug ||
+			'personal-bundle-2y' === plan.wpcom_meta?.product_slug
```

Comment:
I think we can use the `PLAN_PERSONAL` and `PLAN_PERSONAL_2_YEARS` constants here from [constants.js](https://github.com/Automattic/wp-calypso/blob/master/client/lib/plans/constants.js#L27).

Reply:
Good thinking! I may handle this in a small follow-up task, since there are a bunch of these strings in the block below this one where we could make the same adjustment.